### DISCUSSION TITLE
How can i add images to my profile's readme

### DISCUSSION THREAD

**Discussion by Lazar445** [community/community#64634] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How can i add images to my profile's readme please help me how

**Comment by Mr-Proxy-source** [comment]
Just drag and drop image into the page but make sure that you are editing readme.md while doing that!

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
